clients/fossid-webapp/build.gradle.kts

```diff
@@ -30,6 +30,8 @@ plugins {
 dependencies {
     api("com.squareup.retrofit2:retrofit:$retrofitVersion")
 
+    implementation(project(":utils:core-utils"))
```

Comment:
> But I'm not sure yet what to do about the logging...

Shouldn't the logging be part of `common-utils` ?

Reply:
Or use directly in the client log4j.

Reply:
> Or use directly in the client log4j.

Yes, I tend to agree. However, I was wondering whether we should now introduce Log4j here, while there is [an effort going on to get rid of Log4j](https://github.com/oss-review-toolkit/ort/pull/5323). But probably we can still do it, and then get rid of Log4j in a single blow everywhere, instead of creating fragmentation now.

Reply:
@sschuberth the PR has been updated.